apps/pay/next.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   experimental: {
     outputFileTracingRoot: require("path").join(__dirname, "../../"),
     instrumentationHook: true,
+    missingSuspenseWithCSRBailout: false,
```

Comment:
is that really needed?

https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout

`this option will be removed in future version`

Reply:
ie: can't we add a `<Suspense />` wrapper?